src/components/Header.tsx

```diff
@@ -36,7 +36,24 @@ const Header = (props: BoxProps) => {
               fontWeight="black"
               fontSize={{ base: '16px', sm: '36px' }}
             >
-              <RouteLink href="/">Octoclairvoyant</RouteLink>
+              <RouteLink
+                href="/"
+                _hover={{
+                  borderBottomWidth: '4px',
+                  borderBottomColor: 'fuchsia.700',
+                }}
+                _focus={{
+                  boxShadow: 'outline',
+                  borderBottomWidth: '4px',
+                  borderBottomColor: 'fuchsia.700',
+                }}
+                _active={{
+                  borderBottomWidth: '4px',
+                  borderBottomColor: 'fuchsia.900',
+                }}
```

Comment:
I've added some basic `_hover`, `_focus` and `_active` styles.

I think this looks a bit more "on brand" than just a black underline. But maybe you liked the simplicity of the black line, so feel free to revert, or suggest changes. 😉

Reply:
Much better now, thanks! 😄 

I'd like to mention if we can avoid other people than the PR creator pushing changes? It's a little bit intrusive in the sense of I'd have to revert those changes if we are not happy with it, or introduces errors, etc. Also, it can even prevent the PR from being merged _officially_ since some VCS prevent PRs from being approved by people who pushed changes to the branch (we wouldn't be blocked since I have admin permissions, but still).

I think it's better to add a comment with proper suggestions to be pushed by the creator of the PR. What do you think?